GoogleForCreators/web-stories-wp
# Use custom video poster for icon instead of default when available

## Summary
Adjusts the video icon logic to display custom poster when available. Previously, only the default was used.


## Relevant Technical Choices



## To-do



## User-facing changes
|Before|After|
|-----|-----|
| ![Screenshot 2021-06-04 at 14 49 42](https://user-images.githubusercontent.com/3294597/120796991-c4c63b00-c53b-11eb-8a08-5f3c62cbc3ee.png) | ![Screenshot 2021-06-04 at 14 49 14](https://user-images.githubusercontent.com/3294597/120797001-c859c200-c53b-11eb-9682-4445c5124ab8.png) |


## Testing Instructions



### QA






This PR can be tested by following these steps:

1. Add a video element and set as background
2. Set custom poster
3. Verify that in the Filters panel, in the Page Background, and in the Layers panel, the custom poster icon is displayed.

### UAT







This PR can be tested by following these steps:

1.

## Checklist




---



Fixes #7779
